nameko/timer.py

```diff
@@ -50,7 +52,17 @@ def kill(self):
     def _run(self):
         """ Runs the interval loop. """
 
-        sleep_time = self.interval
+        def _process_task():
```

Comment:
rather than adding another function here, maybe just bake this into `handle_timer_tick`. However, we might want to think about the sleep time bug #303